### PR TITLE
Fix structure

### PR DESCRIPTION
**Pull request by MikeAdamczyk**

### CODE REVIEW

File: src/common/Containers/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const MoviesContainer = styled.main`
     grid-template-columns: repeat(auto-fill, 324px);
     grid-auto-rows: 650px;
 
-    @media (max-width: ${({ theme }) => theme.breakpoints.small}px){
+    @media (max-width: ${({ theme }) => theme.breakpoints.small}){
         margin-left: 16px;
         margin-right: 16px;
         margin-bottom: 16px;
```
Comment (anetaszynal) [inline src/common/Containers/index.js:15]: I think you can use the shortcut (margin: 0 16px 16px)

File: src/common/Containers/index.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const PeopleContainer = styled.main`
     grid-template-columns: repeat(auto-fill, 208px);
     grid-auto-rows: 339px;
 
-    @media (max-width: ${({ theme }) => theme.breakpoints.medium}px){
+    @media (max-width: ${({ theme }) => theme.breakpoints.medium}){
```
Comment (anetaszynal) [inline src/common/Containers/index.js:30]: You have two, very similar styles. What do you think to join them ?

**Review by anetaszynal — CHANGES_REQUESTED** [review]
👍🏻

File: src/common/Containers/index.js (hunk 1 above)
Comment (MikeAdamczyk) [reply src/common/Containers/index.js:15]: of course :)

File: src/common/Containers/index.js (hunk 2 above)
Comment (MikeAdamczyk) [reply src/common/Containers/index.js]: done :)

**Review by anetaszynal — APPROVED** [review]

**Pull request merged**